src/external/analysisConverter.ts

```diff
@@ -48,19 +48,24 @@ const logger = Logger('AnalysisConverter');
 const CONVERTER_CONCURRENT_REQUESTS = 5;
 
 const addMetricsToAlignedReadsFile = async (
-  fileDocument: { [k: string]: RdpcFileDocument[] },
+  fileDocument: RdpcFileDocument,
   objectId: string,
-): Promise<RdpcFileDocument[]> => {
-  const runId = fileDocument[objectId][0].analysis.workflow.run_id;
-  const metrics = await getAlignmentMetrics(runId);
+): Promise<RdpcFileDocument> => {
+  const runId = fileDocument.analysis.workflow?.run_id;
```

Comment:
inclined to suggest a destructure here, to simplify legibility, but I may be biased

```suggestion
  const { run_id: runId } = fileDocument.analysis.workflow || {};
```